example/spec/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -120,18 +194,39 @@ describe('When there are granule differences and granule reconciliation is run',
     expect(extraDbUris).toContain(extraFileUri);
   });
 
+  it('generates a report showing number of collections that are in both Cumulus and CMR', () => {
+    // MOD09GQ___006 is in both Cumulus and CMR
+    expect(report.collectionsInCumulusCmr.okCollectionCount).toBeGreaterThan(0);
+  });
+
   it('generates a report showing collections that are in the Cumulus but not in CMR', () => {
     const extraCollection = constructCollectionId(extraCumulusCollection.name.S, extraCumulusCollection.version.S);
     expect(report.collectionsInCumulusCmr.onlyInCumulus).toContain(extraCollection);
+    expect(report.collectionsInCumulusCmr.onlyInCumulus).not.toContain(collectionId);
   });
 
   it('generates a report showing collections that are in the CMR but not in Cumulus', () => {
     // we know CMR has collections which are not in Cumulus
     expect(report.collectionsInCumulusCmr.onlyInCmr.length).toBeGreaterThan(0);
+    expect(report.collectionsInCumulusCmr.onlyInCmr).not.toContain(collectionId);
+  });
+
+  it('generates a report showing number of granules that are in both Cumulus and CMR', () => {
+    expect(report.granulesInCumulusCmr.okGranuleCount).toBeGreaterThan(0);
+  });
+
+  it('generates a report showing granules that are in the Cumulus but not in CMR', () => {
+    expect(report.granulesInCumulusCmr.onlyInCumulus.map((gran) => gran.granuleId)).not.toContain(granuleId);
+  });
+
+  it('generates a report showing granules that are in the CMR but not in Cumulus', () => {
+    // we know CMR has granules which are not in Cumulus (against current stack)
+    expect(report.granulesInCumulusCmr.onlyInCmr.length).toBeGreaterThan(0);
+    expect(report.granulesInCumulusCmr.onlyInCmr.map((gran) => gran.granuleId)).not.toContain(granuleId);
```

Comment:
Could we possibly add to this test to make sure that there is something in there that we expect? Not just check that something is Not in there?

Reply:
We could.  It requires ingest a couple more granules, and the integration-tests run longer.

Reply:
@kkelly51 Added tests for granules `onlyInCmr` and `onlyInCumulus`.